mailbag/parsers/mbox.py

```diff
@@ -0,0 +1,28 @@
+from mailbag.parser import EmailFormatParser
+import mailbox
+
+class Mbox(EmailFormatParser):
+    """Mbox - This concrete class parses mbox file format"""
+    parser_name = 'mbox'
+
+    def parse(self, *args, **kwargs):
+        """Parses the mbox file
+        
+        Args:
+            'file' (key) : The file path of mbox file
+            
+        Example:
+            parse(file="/sample.mbox")
+        """
+        mbox_file = kwargs['file']
+        print("Reading mails from :",mbox_file)        
+        mbox = mailbox.mbox(mbox_file)
+        
+        for key in mbox.iterkeys():
```

Comment:
If we're going to be doing things with the values, rather than `iterkeys()` it might make sense to use `itervalues()` or `iteritems()`:

```python
for key, message in mbox.iteritems():
    ...
```